mocked-api/schemasData.ts

```diff
@@ -1,4 +1,12 @@
 export const schemaCatalogData = [
+  {
+    kind: "ProcessorSchemaEntry",
+    id: "ansible_tower_job_template_sink_0.1",
+    name: "Ansible Tower Job Template",
+    description: "Launch a job template in Ansible Tower.",
+    type: "action",
+    href: "/api/v1/schemas/actions/ansible_tower_job_template_sink_0.1",
```

Comment:
please update the href to be aligned with latest `main`

Reply:
Done.